tests/e2e/config.go

```diff
@@ -122,6 +123,7 @@ type TestConfig struct {
 	timeOffset       time.Duration
 	transformGenesis bool
 	name             string
+	Consumer2ChainID map[ConsumerID]ChainID // dynamic mapping of
```

Comment:
Can we rename to `ConsumerIdToChainId` for better clarity? It would also fit better with `ConsumerIdToChainIdKey`.

Reply:
If this is not used, then why do we have it here? Can't we remove it?